scripts/test/hyriseConsole_test.py

```diff
@@ -38,6 +38,10 @@ def main():
 	console.expect('Loading .*tbl/10_ints.tbl into table "test"')
 	console.expect('Encoding "test" using Unencoded')
 
+	console.sendline("load resources/test_data/bin/float.bin test_bin")
```

Comment:
Not exactly part of this PR, but related: I am wondering, whether this command should really be named import to mirror the export command. load is - internally - only considered for testing via .tbl files.

Reply:
Ack.

Probably open a ticket? Now with the new server (which lacks means to load data), we should consider implement `COPY TABLE x FROM file`.

Reply:
As you suggested, I opened a separate ticket: https://github.com/hyrise/hyrise/issues/1933